packages/hardhat-core/src/internal/cli/analytics.ts

```diff
@@ -150,8 +148,8 @@ export class Analytics {
       // https://support.google.com/tagmanager/answer/6164990
       //
       // Custom dimension 1: Project Id
-      // This is computed as the keccak256 hash of the project's absolute path.
-      cd1: this._projectId,
+      // TODO-HH: Can we remove this?
```

Comment:
I don't think we can. Debugging these things is super cumbersome also.

Reply:
Removed the comment and used "hardhat-project" instead of "project" as the first dimension.